.sonarqube/sonar-scanner.properties

```diff
@@ -14,14 +14,13 @@
 
 # Configuration file for Sonar Scanner used for CI
 
-sonar.projectKey=usdot-fhwa-stol_CARMAWebUi
+sonar.projectKey=usdot-fhwa-stol_carma-web-ui
 sonar.organization=usdot-fhwa-stol
 sonar.cfamily.cache.enabled=false
 sonar.cfamily.compile-commands=/opt/carma/build/compile_commands.json
 sonar.host.url=https://sonarcloud.io
-sonar.sources=website/scripts/, website/utils
+sonar.sources=website/
 # Set Git as SCM sensor
 sonar.scm.disabled=false
```

Comment:
Can we remove this sonar.scm.enabled=true if disabled is false?

Reply:
Done